chromium_src/chrome/browser/ui/omnibox/omnibox_theme.cc

```diff
@@ -42,7 +45,7 @@ const SkColor kPrivateLocationBarBackground = SkColorSetRGB(0x1b, 0x0e, 0x2c);
 }
 
 // Overriden version
-SkColor GetOmniboxColor(OmniboxPart part,
+SkColor GetOmniboxColor_ChromiumImpl(OmniboxPart part,
```

Comment:
What calls this?

Reply:
The return statement on line 95 below seems to call this recursively? 
Seems like `#define GetOmniboxColor GetOmniboxColor_ChromiumImpl` should have been left along with this function name?

Reply:
yep, this looks wrong to me. Nice catch

Reply:
ya something seems not right here, I'll test. Thanks.

Reply:
@mkarolin this is fixed now, could you take a look?

Reply:
LGTM